pandas/core/series.py

```diff
@@ -3688,7 +3688,11 @@ def f(x):
 
         if len(mapped) and isinstance(mapped[0], Series):
             from pandas.core.frame import DataFrame
-            return DataFrame(mapped.tolist(), index=self.index)
+            # GH 25959 we don't need to call tolist
+            # if we've been given an ExtensionArray
+            if not isinstance(mapped, ExtensionArray):
```

Comment:
Use is_extension_array_dtype(mapped.dtype)

Why do we call tolist in the first place?

Reply:
It was initially there - I do not know, I resolved your problem though.